Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogHeaderWidget.cpp

```diff
@@ -272,8 +280,13 @@ namespace O3DE::ProjectManager
         m_downloadingListWidget->show();
     }
 
-    void GemCartWidget::GemDownloadRemoved(const QString& gemName)
+    void GemCartWidget::ObjectDownloadRemoved(const QString& gemName, DownloadController::DownloadObjectType objectType)
     {
+        if (objectType != DownloadController::DownloadObjectType::Gem)
+        {
+            return;
+        }
```

Comment:
not that I'm advocating creating an EBUS for this, but this is a case where an addressable event bus is useful - i.e. GemCartWidget could listen for download events for gem type objects only and not have to check every download event to make sure it is for a gem type.

Reply:
How about an AZ Event instead, where the GemCartWidget provides a handler to a function on the GemCatalogHeaderWidget which registers the handler with an event that is signaled whever an a gem object is removed?

The Settings Registry [notify event](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/Settings/SettingsRegistry.h#L184-L192) can be used as an example of using an AZ::Event for notifications.

Reply:
I like this suggestion. This will be good in a follow up PR so this can unblock some other work and these changes don't get lost in what has already become a large PR.